tide/master/ui/MasterWindow.h

```diff
@@ -62,7 +62,7 @@ class MasterWindow : public QMainWindow
 public:
     /** Constructor. */
     MasterWindow(DisplayGroupPtr displayGroup, OptionsPtr options,
-                 MasterConfiguration& config);
+                 MasterConfiguration& config, LockPtr lock);
```

Comment:
For readability group displaygroup, options and lock parameters, leaving MasterConfiguration at the end.